src/nifake/metadata/attributes.py

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8 -*-
-# This file is generated
+# This file is generated from NI-FAKE API metadata version 0.2.0d23
 attributes = {
     1000000: {
         'access': 'read-write',
         'channel_based': False,
         'documentation': {
             'description': 'An attribute of type bool with read/write access.'
         },
+        'lv_property': 'Fake attributes:Read Write Bool',
```

Comment:
I had never seen this. Why is this part of the nimi-python metadata?

Reply:
Answered: it is used in documentation to say what the corresponding LabVIEW property is.